docs/source/development/linting.md

```diff
@@ -1,33 +1,34 @@
-# Linting
+# Code formatting and linting
 
 ## Introduction
-Linting tools are used to improve the quality of your code by checking for errors and making sure it meets a stylistic
-standard before it is run. As a project grows and goes through various stages of development, it becomes important to
-maintain the code quality. Linting makes your code more readable, easy to debug and maintain, and stylistically
-consistent.
 
-## Set up linting tools
-There are a variety of linting tools available to use with your Kedro projects. This guide shows you how to use
+Code formatting guidelines set a standard for the layout of your code, for stylistic elements such as use of line breaks and whitespace. Format doesn't have any impact on how the code works, but using a consistent style makes your code more readable, and makes it more likely to be reused.
+
+Linting tools check your code for errors such as a missing bracket or line indent. This can save time and frustration because you can catch errors in advance of running the code.
+
+As a project grows and goes through various stages of development it becomes important to maintain code quality. Using a consistent format and linting your code ensures that it is consistent, readable, and easy to debug and maintain.
+
+## Set up Python tools
+There are a variety of Python tools available to use with your Kedro projects. This guide shows you how to use
 [`black`](https://github.com/psf/black), [`flake8`](https://github.com/PyCQA/flake8), and
-[`isort`](https://github.com/PyCQA/isort) to lint your Kedro projects.
+[`isort`](https://github.com/PyCQA/isort) to format and lint your Kedro projects.
 - **`black`** is a [PEP 8](https://peps.python.org/pep-0008/) compliant opinionated Python code formatter. `black` can
 check for styling inconsistencies and reformat your files in place.
 [You can read more in the `black` documentation](https://black.readthedocs.io/en/stable/).
 - **`flake8`** is a wrapper around [`pep8`](https://pypi.org/project/pep8/),
```

Comment:
Wondering if it's already time to start thinking about replacing flake8 by ruff 👀 https://pypi.org/project/ruff/ cc @datajoely

Reply:
so my understanding is that the majority of projects have adopted it in pre-commit rather than full CI as it doesn't yet have parity with Pylint / flake8